site/scripts/create-db-indexes.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/first */
+import dotenv from "dotenv";
+
+dotenv.config({ path: ".env.local" });
```

Comment:
Let’s use `yarn exe ...` – it’s in `main` already. If you want to keep the script inside `site`, we can create a copy of `exe` in a local `package.json`. This will save us from using `dotenv` here and repeating node options in `package.json`. We won’t have to add a new `package.json` script per each script file too, so it will be easier to maintain the growing number of scripts.

Reply:
Sure that sounds good. Will get around to it soon! Thank you a lot :)